### PR TITLE
python-docx2python: added package

### PR DESCRIPTION
**Pull request by D3vil0p3r**
Used for LinkScope tool to avoid the usage of Python venv: https://github.com/BlackArch/blackarch/issues/3659

### CODE REVIEW

File: packages/python-docx2python/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-docx2python
+_pkgname=docx2python
```
Comment (noraj) [inline packages/python-docx2python/PKGBUILD:5]:
```suggestion
_pkgname=${pkgname#python-}
```

File: packages/python-docx2python/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+pkgver=2.0.5
+pkgrel=1
+pkgdesc='Extract docx headers, footers, text, footnotes, endnotes, properties, and images to a Python object.'
+arch=('any')
+url='https://pypi.org/project/docx2python'
```
Comment (noraj) [inline packages/python-docx2python/PKGBUILD:10]:
```suggestion
url='https://github.com/ShayHill/docx2python'
```

File: packages/python-docx2python/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+license=('MIT')
+depends=('python' 'python-lxml')
+makedepends=('python-setuptools')
+options=(!emptydirs)
+source=("https://files.pythonhosted.org/packages/source/${_pkgname::1}/$_pkgname/$_pkgname-$pkgver.tar.gz")
+sha512sums=('SKIP')
```
Comment (noraj) [inline packages/python-docx2python/PKGBUILD:16]: dont skip, skip is only for git

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/python-docx2python/PKGBUILD (hunk 3 above)
Comment (D3vil0p3r) [reply packages/python-docx2python/PKGBUILD]: Applied SHA512 hash

**Pull request merged**